pkg/rbd/rbd_util.go

```diff
@@ -30,6 +35,7 @@ import (
 
 const (
 	imageWatcherStr = "watcher="
+	rbdImageFormat1 = "1"
```

Comment:
Image format v1 is long-since deprecated -- don't allow users to attempt to create them since it will just fail.